src/dsp_tools/utils/xmlupload/upload_stashed_xml_texts.py

```diff
@@ -43,265 +42,82 @@ def log_unable_to_retrieve_resource(
 
 def _log_unable_to_upload_xml_resource(
     received_error: BaseError,
-    stashed_resource: XMLResource,
-    all_link_props: XMLProperty,
+    stashed_resource_id: str,
+    prop_name: str,
 ) -> None:
     """
     This function logs if it is not possible to upload a xml resource.
 
     Args:
         received_error: Error received
-        stashed_resource: resource that is stashed
-        all_link_props: all the link properties from that resource
+        stashed_resource_id: id of the resource
+        prop_name: name of the property
     """
     # print the message to keep track of the cause for the failure
     # apart from that; no action is necessary:
     # this resource will remain in nonapplied_xml_texts, which will be handled by the caller
     orig_err_msg = received_error.orig_err_msg_from_api or received_error.message
-    err_msg = f"Unable to upload the xml text of '{all_link_props.name}' of resource '{stashed_resource.id}'."
+    err_msg = f"Unable to upload the xml text of '{prop_name}' of resource '{stashed_resource_id}'."
     print(f"    WARNING: {err_msg} Original error message: {orig_err_msg}")
     logger.warning(err_msg, exc_info=True)
 
 
-def _log_iri_does_not_exist_error(
-    received_error: KeyError,
-    stashed_resource: XMLResource,
-    all_link_props: XMLProperty,
-) -> None:
+def _log_iri_does_not_exist_error(res_id: str, prop_name: str) -> None:
```

Comment:
my preference is to have every parameter on one line. With this one-line-formatting, you never see how many parameters there are. It's really not well visible.

Reply:
here I do not agree - turning a one-liner into 4-5 lines bloats the code a lot. And while github isn't so nice about it, in the IDE, syntax highlighting makes the arguments perfectly distinct.

in a case where there is a long list of arguments and something "important" happens, I'm happy to use one line per parameter and named arguments; but this kind of helper really only serves the purpose of hiding implementation details in a way that shouldn't get in the way, so it should be as "discrete" as possible.  
Also, where the function is called, you should not be concerned with the arguments that are passed, unless you focus on it - but on the higher level of abstraction of the caller side, all you should care about is the function name `log_XYZ_error` to know "ah, here the error is logged" and continue. The detail of what arguments the logging requires should not concern you on this level of abstraction. (On the function definition side, this is of course different.)

Before causing you to dislike the newly written code, I may be convinced otherwise. But I would much prefer not having "hard rules" on code style, but to have "guide rails" that allow for this kind variation. What do you think?